src/handwritten/_install_daqmx.py

```diff
@@ -117,49 +152,89 @@ def _multi_access_temp_file(
                 ) from e
 
 
-def _load_data(json_data: str) -> Tuple[Optional[str], Optional[str]]:
+@contextlib.contextmanager
+def _multi_access_temp_folder(*, delete: bool = True) -> Generator[str, None, None]:
+    """
+    Context manager for creating a temporary folder.
+
+    """
+    try:
+        temp_folder = tempfile.TemporaryDirectory()
+        _logger.debug("Created temp folder: %s", temp_folder.name)
+    except Exception as e:
+        _logger.info("Failed to create temporary folder.", exc_info=True)
+        raise click.ClickException(
+            f"Failed to create temporary folder '{temp_folder.name}'.\nDetails: {e}"
+        ) from e
+
+    try:
+        yield temp_folder.name
+    finally:
+        if delete:
```

Comment:
TemporaryDirectory supports optional delete: `tempfile.TemporaryDirectory(delete=delete)`

Reply:
ok, this is the default